compiler/rustc_target/src/spec/targets/xtensa_esp32s3_none_elf.rs

```diff
@@ -0,0 +1,24 @@
+use crate::spec::{base::xtensa, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "xtensa-none-elf".into(),
+        pointer_width: 32,
+        data_layout: "e-m:e-p:32:32-v1:8:8-i64:64-i128:128-n32".into(),
+        arch: "xtensa".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: None,
+            tier: None,
+            host_tools: None,
+            std: None,
+        },
```

Comment:
Can you complete the metadata for each of these targets? It's only `Option` for the targets we've not populated these for yet.

Reply:
Hi! I think I just added it, but CI is still failing :/